raiden-cli/src/utils/formatting.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { RaidenChannel } from 'raiden-ts';
+import { ApiChannel } from '../types';
+
+export function transformChannelFormatSdkToApi(channel: RaidenChannel): ApiChannel {
+  return {
+    channel_identifier: channel.id ?? 0, // TODO: how can this be undefined?
```

Comment:
As we discussed, this was a leftover of when channel state could be `opening` and no `id` would be available then. Nowadays, first channel state is `open`, and id,  settleTimeout, etc, are always defined. I'll fix.

Reply:
:heavy_check_mark: